requirements-python.txt

```diff
@@ -25,3 +21,4 @@ click-completion==0.5.0
 PTable==0.9.2
 pycrypto==2.6.1
 pyzmq==18.0.1
+tenacity==5.0.4
```

Comment:
what is tenacity used for?

Reply:
this is a bit more modern retrying package, that I used to replace the wait_for